reconcile/cli.py

```diff
@@ -883,11 +883,38 @@ def jenkins_webhooks_cleaner(ctx):
 
 
 @integration.command(short_help="Validate permissions in Jira.")
+@click.option(
+    "--exit-on-permission-errors/--no-exit-on-permission-errors",
+    help="Throw and error in case of board permission errors. Useful for PR checks.",
+    default=False,
+)
```

Comment:
this should be inherent to the integration. all integration exit upon error.

Reply:
The idea is that the integration doesn't fail for PR checks and during the runtime (reconcile loop) as long as we don't have admin permissions on all boards. this will make the transition much smoother.

Reply:
BTW, this is just for lack of admin permissions and temporary. All other errors will raise an `exit 1` as usual.